azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDKWrapper.kt

```diff
@@ -19,14 +17,38 @@ import com.azure.android.core.util.RequestContext
 import java9.util.concurrent.CompletableFuture
 
 internal class ChatSDKWrapper(
-    private val instanceId: Int,
-    private val context: Context,
+    val context: Context,
+    chatConfig: ChatConfiguration,
 ) : ChatSDK {
 
-    private val chatConfig get() = ChatCompositeConfiguration.getConfig(instanceId).chatConfig!!
+    private val chatAsyncClient by lazy {
+        ChatClientBuilder()
+            .endpoint(endPointURL)
+            .credential(credential)
+            .addPolicy(
+                UserAgentPolicy(
+                    applicationID,
+                    sdkName,
+                    sdkVersion
+                )
+            )
+            .buildAsyncClient()
```

Comment:
ing logicWe may want to have error handling logic for this code. Having in function will help.